inputfiles/idl/HTML - Tabular data.commentmap.json

```diff
@@ -0,0 +1,25 @@
+{
+    "table-caption": "Returns the table's caption element.\nCan be set, to replace the caption element.",
```

Comment:
comments do not seem to be showing here either.

Reply:
Same here, it's not `htmltableelement-caption` so it doesn't show up.